### DISCUSSION TITLE
Is it possible to automatically update the dns blacklist ?

### DISCUSSION THREAD

**Discussion by pallepop** [hassio-addons/addon-adguard-home#296] [Q&A]
Hello,

I looked through the documentation and the home assistant services but couldn't find a way to automatically update the dns blacklist.
I Think it would be a great feature if it's not already there.

Still, thank you for this awesome add-on !

Cheers

**Comment by frenck** [comment]
> I looked through the documentation and the home assistant services but couldn't find a way to automatically update the dns blacklist.

That is about the Home Assistant integration, however, this is not the repository for that. This is the repository of the add-on. Add-ons do not provide Home Assistant services or integrations.

I think you are mixing up things.

**Reply by pallepop** [reply]
Ok, thought it was the right place to ask but i'm no home assistant developper :p. 

I guess i should go to the adguard repository then ?

Ty for answering

**Reply by AlecRust** [reply]
By default AdGuard Home will refresh the filter lists every 24 hours, as-per settings.

![image](https://user-images.githubusercontent.com/247634/150552625-2e897681-f827-4fd8-bc36-a8cf6306de98.png)

**Reply by pallepop** [reply]
Now i feel dumb, didn't think to look through settings while trying every other complex way to do it...

Cheers mate !
